cuegui/setup.py

```diff
@@ -57,5 +57,12 @@
         ]
     },
     test_suite='tests',
+    install_requires=[
+        'PySide2',
```

Comment:
Let's alphabetize this list.

Reply:
it's ASCIIbetical which is the order that PyCharm presents by default 😄 I’ll adjust to proper alphabetical